whylog/config/rule.py

```diff
@@ -125,9 +135,12 @@ def _create_constraints_list(cls, parser_ids_mapper, user_rule_intent):
 
     @classmethod
     def from_dao(cls, serialized_rule, parsers):
+        # TODO: restore serialized_rule["linkage"] when UserRuleIntent will support rule linkage
         return Rule(
-            [parsers[cause] for cause in serialized_rule["causes"]],
-            parsers[serialized_rule["effect"]], serialized_rule["constraints"]
+            [
+                parsers[cause] for cause in serialized_rule["causes"]
+            ], parsers[serialized_rule["effect"]], serialized_rule["constraints"],
+            serialized_rule.get("linkage", Rule.LINKAGE_AND)
```

Comment:
I don't think we need to keep support for old style of rules where linkage was not specified.